site/src/components/form.md

```diff
@@ -43,58 +43,216 @@ It can be used to hold `.field-body` containers, which holds the individual elem
+			>
+				<option value="1">1</option>
+				<option value="2">2</option>
+				<option value="3">3</option>
+			</select>
+		</div>
+	</div>
+	<div class="field">
+		<label class="label" for="sample-tags">
+			Tags
+			<span class="required-indicator"></span>
+		</label>
+		<span id="sample-tags-description" class="field-description"> Select up to 5 tags. </span>
+		<div class="field-body">
+			<input
+				id="sample-tags"
+				name="tags"
+				class="tag-input input"
+				aria-describedby="sample-select-description"
+				hidden
+				aria-hidden="true"
+				value=""
+				mintags="1"
+				maxtags="5"
+			/>
+			<div class="tag-input-holder margin-bottom-xxs has-height-auto">
+				<div class="tags"></div>
+				<input
+					role="combobox"
+					maxlength="100"
+					id="tag-input"
+					class="autocomplete-input input"
+					type="text"
+					aria-label="Tag selector"
+					placeholder="Enter tags..."
+					hidden
+				/>
+			</div>
+			<button class="add-tags-button" type="button">Add tags</button>
+		</div>
+	</div>
+	<div class="field">
+		<label class="label" for="sample-text-area">
+			Textarea
+			<span class="required-indicator"></span>
+		</label>
+		<div class="field-body">
+			<textarea
+				id="sample-text-area"
+				name="text-area"
+				class="textarea"
+				minlength="10"
+				rows="4"
+				cols="30"
+			>
+			</textarea>
+		</div>
+	</div>
+
+	<div class="field">
+		<div class="field-body">
+			<star-rating id="star-rating-1" name="star-rating-1" required>
+				<legend slot="legend">Star rating</legend>
+				<span slot="label-1">Terrible</span>
+				<span slot="label-2">Poor</span>
+				<span slot="label-3">Fair</span>
+				<span slot="label-4">Good</span>
+				<span slot="label-5">Great</span>
+			</star-rating>
+		</div>
 	</div>
 	<div class="display-flex">
-		<button class="button button-primary margin-right-xxs">Submit</button>
-		<button class="button button-danger">Cancel</button>
+		<button type="submit" class="button button-primary button-filled">Submit form</button>
+		<button type="button" class="button">Cancel</button>
 	</div>
 </form>
 ```
 
-### Required Indicator
-
-For control elements that are required for form submission, you can add a span element with the `.required-indicator` class inside the label or adjacent to the element.
+##### Simple form without edits
 
 ```html
-<form>
-	<p class="\visually-hidden">Required fields are marked with asterisk/star</p>
-	<fieldset class="field">
-		<legend class="field-label">Radio group <span class="required-indicator"></span></legend>
-		<div class="field-body">
-			<label class="radio">
-				<input checked name="question-1" type="radio" class="radio-dot" value="Yes" required />
-				<span class="radio-label-text">Yes</span>
-			</label>
-			<label class="radio">
-				<input checked name="question-1" type="radio" class="radio-dot" value="No" />
-				<span class="radio-label-text">No</span>
-			</label>
-		</div>
-	</fieldset>
+<form
```

Comment:
I added this example to demonstrate the usage of the "new" attribute, but button values aren't automatically included in formData. It's not fully functional so it may not be the best example to show and may be better to remove it. Otherwise, I'll need to add more custom logic to make it work properly.